src/python/turicreate/test/test_sframe_s3.py

```diff
@@ -23,7 +23,7 @@
 
 
 @pytest.mark.skipif(
-    os.environ.get("TURI_ENABLE_SF_S3") is None,
+    os.environ.get("TC_ENABLE_S3_TESTS", "0") == "1",
```

Comment:
Doesn't this skip it if it's 1?

Reply:
do you set that to 1 to skip? I though 1 means to turn it on.

Reply:
ah, may bad.